Formula/tasksh.rb

```diff
@@ -1,6 +1,6 @@
 class Tasksh < Formula
   desc "Shell wrapper for Taskwarrior commands"
-  homepage "https://tasktools.org/projects/tasksh.html"
+  homepage "https://web.archive.org/web/20170227160805/https://tasktools.org/projects/tasksh.html"
```

Comment:
let's use this url, seems the same content as the web archive page.
```suggestion
  homepage "https://gothenburgbitfactory.org/projects/tasksh.html"
```

Reply:
Ah, good find.